app/src/main/java/org/greenstand/android/TreeTracker/view/TextButton.kt

```diff
@@ -171,26 +163,26 @@ fun CustomDialog(
             .fillMaxWidth()
             .wrapContentHeight()
             .padding(2.dp)
-            .border(1.dp, color = Color(0xFF75B926), shape = RoundedCornerShape(percent = 10))
+            .border(1.dp, color = AppColors.Green, shape = RoundedCornerShape(percent = 10))
             .clip(RoundedCornerShape(percent = 10)),
-        backgroundColor = Color(0XFF191C1F),
+        backgroundColor = AppColors.Gray,
         text = {
             Column(
                 modifier = Modifier
                     .fillMaxWidth()
                     .wrapContentHeight()
             ) {
-
                 Text(
                     text = content,
                     color = CustomTheme.textColors.primaryText,
                     style = CustomTheme.typography.regular,
                     modifier = Modifier.padding(bottom = 5.dp)
                 )
-                if (isTextInputAvailable) {
+                onTextInputValueChange?.let {
```

Comment:
<nit> when a 'let' block is large like this 'it' can lose context. I'd suggest to give 'it' a name since it's used 4 lines below.